client/components/Register.jsx

```diff
@@ -37,35 +31,63 @@ function Register() {
   }
 
   const handleSubmit = (evt) => {
+    const gravatar = `https://www.gravatar.com/avatar/${md5(
+      user.email
+    )}?d=retro&f=y`
+
     evt.preventDefault()
     const userInfo = {
       auth0Id: user.auth0Id,
+      email: user.email,
+      image: gravatar,
       ...form,
     }
     addUser(userInfo, user.token)
       .then(() => dispatch(updateLoggedInUser(userInfo)))
       .catch((err) => setErrorMsg(err.message))
   }
 
-  const hideError = () => {
-    setErrorMsg('')
-  }
-
   return (
     <>
       <h2>Complete profile set up</h2>
-      {errorMsg && <Error onClick={hideError}>Error: {errorMsg}</Error>}
-      <GridForm onSubmit={handleSubmit}>
-        <ColOne htmlFor="username">Username:</ColOne>
-        <ColTwoText
+      <form>
+        <TextField
           type="text"
-          id="username"
-          name="username"
-          value={form.username}
+          id="name"
+          name="name"
+          label="Name"
+          value={form.name}
           onChange={handleChange}
         />
-        <Button disabled={!form.username}>Save Profile</Button>
-      </GridForm>
+        {errorMsg ? (
+          <TextField
```

Comment:
Rather than rendering 2 TextFields just have the ternary on the error pop.

Reply:
Changed to `error={errorMsg ? true : false}`

Reply:
Could simplify it even more by saying 'error={Boolean(errorMsg)}'